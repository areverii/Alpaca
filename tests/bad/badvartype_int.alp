function void main(<string> args):
    int x
    x = 1
    not x
end
