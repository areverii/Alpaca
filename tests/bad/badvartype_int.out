Fatal error: exception Failure("illegal unary operator not int in not x")
